README.md

```diff
@@ -159,6 +159,120 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
 
 The resulting model will be saved to the `hf-ner-model/` directory.
 
+## `terms.openai.fetch`: Fetch phrases and terms based on a query
+
+This recipe generates terms and phrases entity predictions obtained from a large language model. These
+terms can be curated and turned into patterns files, which can help with downstream annotation tasks. 
+
+```bash
+python -m prodigy terms.openai.fetch query filepath [--options] -F ./recipes/openai_terms.py
+```
+
+|        Argument       | Type  | Description                                         | Default                         |
+|:---------------------:|-------|-----------------------------------------------------|---------------------------------|
+| `query`               | str   | Query to send to OpenAI                             |                                 |
+| `output_path`         | Path  | Path to save the output                             |                                 |
+| `--seeds`,`-s`        | str   | One or more comma-seperated seed phrases.           | `""`                            |
+| `--n`,`-n`            | int   | Number of items to generate                         | `100`                           |
+| `--model`, `-m`       | str   | GPT-3 model to use for completion                   | `"text-davinci-003"`            |
+| `--prompt-path`, `-p` | Path  | Path to jinja2 prompt template                      | `templates/terms_prompt.jinja2` |
+| `--verbose`,`-v`      | bool  | Print extra information to terminal                 | `False`                         |
+| `--resume`, `-r`      | bool  | Resume by loading in text examples from output file | `False`                         |
+| `--progress`,`-pb`    | bool  | Print progress of the recipe.                       | `False`                         |
+| `--temperature`,`-t`  | float | OpenAI temperature param                            | `1.0`                           |
+| `--top-p`, `--tp`     | float | OpenAI top_p param                                  | `1.0`                           |
+| `--best-of`, `-bo`    | int   | OpenAI best_of param"                               | `10`                            |
+| `--n-batch`,`-nb`     | int   | OpenAI batch size param                             | `10`                            |
+| `--max-tokens`, `-mt` | int   | Max tokens to generate per call                     | `100`                           |
+
+### Example usage
+
+Suppose you're interested in detecting skateboard tricks in text, then you might want to start
+with a term list of known tricks. You might want to start with the following query:
+
+```bash
+# Base behavior, fetch at least 100 terms/phrases
+python -m prodigy terms.openai.fetch "skateboard tricks" tricks.jsonl --n 100 -F recipes/openai_terms.py
```

Comment:
You should probably specify the template on the command line in this example. If run as is, it uses the variation template that is hardcoded as default in the current version of the recipe.